src/Markup/Avalonia.Markup.Xaml/Converters/AvaloniaPropertyTypeConverter.cs

```diff
@@ -2,85 +2,80 @@
 // Licensed under the MIT license. See licence.md file in the project root for full license information.
 
 using System;
+using System.ComponentModel;
 using System.Globalization;
-using System.Linq;
+using System.Text.RegularExpressions;
+using Avalonia.Markup.Xaml.Templates;
+using Avalonia.Styling;
+using Portable.Xaml;
+using Portable.Xaml.ComponentModel;
+using Portable.Xaml.Markup;
 
 namespace Avalonia.Markup.Xaml.Converters
 {
-    using Avalonia.Styling;
-    using Portable.Xaml;
-    using Portable.Xaml.ComponentModel;
-    using System.ComponentModel;
-    using Portable.Xaml.Markup;
-
     public class AvaloniaPropertyTypeConverter : TypeConverter
     {
+        private static readonly Regex regex = new Regex(@"^\(?(\w*)\.(\w*)\)?|(.*)$");
```

Comment:
This allows attached properties to be optionally specified with braces surrounding the property, as in a standard `Binding`.

Reply:
Is there a way we can do this without using regular expressions? Regex uses lot of memory for us just checking if the string starts and ends with parentheses.